### PR TITLE
Bug 1239674: fix unnecessary screen brightness transition in cancelin…

### PR DESCRIPTION
**Pull request by cervantesyu**
…g screen timeout. r=timdream

### CODE REVIEW

**Comment by autolander** [comment]
Autolander could not find a bug number in your pull request title. All pull requests should be in the format of: Bug [number] - [description].

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=954bcf81b6338ae8dd3640e2a507f01dda7603f4

File: apps/system/js/screen_auto_brightness.js [hunk 1]
```diff
@@ -23,8 +23,15 @@
   ScreenAutoBrightness.prototype.SYNTHETIC_BRIGHTNESS_DELAY = 250;
 
   ScreenAutoBrightness.prototype._previousBrightnessValues = null;
-  // Start with half of max brightness.
-  ScreenAutoBrightness.prototype._currentBrightness = 0.5;
+
+  ScreenAutoBrightness.prototype._currentBrightness = undefined;
+  ScreenAutoBrightness.prototype.currentBrightness = function() {
```
Comment (timdream) [inline apps/system/js/screen_auto_brightness.js]: Should this be `getCurrentBrightness`?

**Comment by autolander** [comment]
Autolander could not find a bug number in your pull request title. All pull requests should be in the format of: Bug [number] - [description].

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=4358a635bbe843fa37b48ff6ef7631e58f1beb30

**Comment by autolander** [comment]
Autolander could not find a bug number in your pull request title. All pull requests should be in the format of: Bug [number] - [description].

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=daffcc8735d8240dfd5bc94c63ed95a32e328590

**Comment by autolander** [comment]
Autolander could not find a bug number in your pull request title. All pull requests should be in the format of: Bug [number] - [description].

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=f53c17e3abe806ec6647cccfd26b55531f1a5e5f

**Pull request merged**